starport/services/network/join_test.go

```diff
@@ -0,0 +1,370 @@
+package network
```

Comment:
The readability is really hard in this file and some others.

Can we use line breaks between different set of logic, multi line structs an also this format:

```go
myvar
    .On()
    .Return()
```